src/components/Modals/CourseMenu.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="courseMenu">
+  <div v-bind:class="{'courseMenu-movedNotCompact': isA , 'courseMenu': isB, 'courseMenu-movedCompact':isC}" class="courseMenu" >
```

Comment:
Can you rename "isA" "isB" and "isC" to something more descriptive? Had a hard time understanding exactly what this code does for a while.